cms/src/test/java/uk/nhs/digital/apispecs/IsAnyTrueHelperTest.java

```diff
@@ -107,21 +109,39 @@ public void throwExceptionWhenCollectionIsNull() throws IOException {
     }
 
     @Test
-    public void throwExceptionWhenCollectionIsNotBoolean() throws IOException {
+    public void throwExceptionWhenAtLeastOneIsNull() throws IOException {
+        Boolean[] params = {false, null};
+        StubOptions options = new StubOptions(params);
+
+        try {
+            // when
+            isAnyTrueHelper.apply(item,options);
+
+        } catch (Exception e) {
+
+            // then
+            assertThat("Exception in execution",
+                e.getMessage(),
+                containsString("All parameters are required to be boolean but at least one provided value was not.")
+            );
+        }
+    }
 
-        Object[] params = {false,"test", 4};
+    @Test
+    public void throwExceptionWhenAtLeastOneIsNonBoolean() throws IOException {
```

Comment:
As mentioned in earlier comments and discussed on Slack, this test should fail when the method under test (`isAnyTrueHelper.apply`) fails to throw any exceptions (and the one we're asserting against in particular).

Reply:
They are failing. They don't need to be failed explicitly.

Reply:
See my comment to `throwExceptionWhenAtLeastOneIsNull`.

Reply:
Fixed